CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+## [2.0.1] - 2019-06-10
```

Comment:
```suggestion
```

Let's avoid bumping the version in a PR like this, as I believe that change needs to come at the time of cutting the release.